From: Werner Koch <wk@gnupg.org>
Date: Sat, 3 Dec 2016 21:35:45 +0100
Subject: gpg: Fix error code arg in ERRSIG status line.

* g10/mainproc.c (check_sig_and_print): Use gpg_err_code to return an
error code in ERRSIG.
--

Debian-bug-id: 846834
Signed-off-by: Werner Koch <wk@gnupg.org>
(cherry picked from commit ef10c348bffc7dad19e1832bebc453755d209420)
---
 g10/mainproc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/g10/mainproc.c b/g10/mainproc.c
index c1819f0..314d515 100644
--- a/g10/mainproc.c
+++ b/g10/mainproc.c
@@ -2218,7 +2218,7 @@ check_sig_and_print (CTX c, kbnode_t node)
       snprintf (buf, sizeof buf, "%08lX%08lX %d %d %02x %lu %d",
                 (ulong)sig->keyid[0], (ulong)sig->keyid[1],
                 sig->pubkey_algo, sig->digest_algo,
-                sig->sig_class, (ulong)sig->timestamp, rc);
+                sig->sig_class, (ulong)sig->timestamp, gpg_err_code (rc));
       write_status_text (STATUS_ERRSIG, buf);
       if (gpg_err_code (rc) == GPG_ERR_NO_PUBKEY)
         {
